tools/generate-helpers/pg-table-bindings/store.go.tpl

```diff
@@ -366,7 +374,25 @@ func (s *storeImpl) UpsertMany(ctx context.Context, objs []*{{.Type}}) error {
     } else if !ok {
         return sac.ErrResourceAccessDenied
     }
-    {{- end}}
+    {{- else if .Obj.IsDirectlyScoped -}}
+    {{ template "defineScopeChecker" "READ_WRITE" }}
+    if ok, err := scopeChecker.Allowed(ctx); err != nil {
+        return err
+    } else if !ok {
+        for _, obj := range objs {
+            {{- if .Obj.IsClusterScope }}
+            subScopeChecker := scopeChecker.ClusterID({{ "obj" | .Obj.GetClusterID }})
+            {{- else if .Obj.IsNamespaceScope }}
+            subScopeChecker := scopeChecker.ClusterID({{ "obj" | .Obj.GetClusterID }}).Namespace({{ "obj" | .Obj.GetNamespace }})
+            {{- end }}
+            if ok, err := subScopeChecker.Allowed(ctx); err != nil {
+                return err
+            } else if !ok {
+                return sac.ErrResourceAccessDenied
```

Comment:
Why do we not return the allowed resources?

Reply:
So you would like to get a list of all resources that access was denied?